_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -28,13 +28,13 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any non-focusable `iframe` element that has [focusable][] content.
+This rule applies to any `iframe` element that [contains](#akn7bn:contain) elements that are [visible][] and part of the [sequential focus navigation][].
+
+An element is <dfn id="akn7bn:contain">contained</dfn> in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][].
 
 ## Expectation
 
-For each test target, the [nested browsing context][] does not [contain](#akn7bn:contain) elements that are [visible][] and part of the [sequential focus navigation][].
-
-An element is <dfn id="akn7bn:contain">contained</dfn> in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][].
+For each test target, the [nested browsing context][] does not have a negative number as a `tabindex` [attribute value][].
```

Comment:
Agree with @giacomo-petri The negative tabindex is set on the `iframe` element, aka the browsing context container, not on the nested browsing context (which is an abstract notion not really "existing" in the HTML code and not having attributes).

```suggestion
None of the test target has a negative number as a `tabindex` [attribute value][].
```